src/renderer/screens/Editor/ImportExportDialog.js

```diff
@@ -26,18 +28,37 @@ export const ImportExportDialog = withSnackbar(props => {
           null,
           2
         );
-  const onConfirm = () => {
+
+  function onConfirm() {
```

Comment:
Any reason for going from `const onConfirm = () => { ` to `function onConfirm() {`? (Apart from the latter being shorter)

Reply:
In this particular circumstance, it doesn't really matter. I have always tended to lean towards the classic function declaration unless dealing with issues of `this` due to function hoisting, no arguments array in arrow functions, and in some browsers (which shouldn't be an issue in this case with Electron) arrow functions don't actually get their names in the stack trace.

Happy to go either way here though.

Reply:
Lets leave them as functions then.